### PR TITLE
fix: Refactors code and design improvements for filters

### PR DESCRIPTION
**Pull request by nithindavid**
**This PR will**
- Changes overall design of filters
- Increases size of parent modal
- refactors CSS code

image

### CODE REVIEW

File: app/javascript/dashboard/components/widgets/conversation/components/FilterInput.vue [hunk 1]
```diff
@@ -1,40 +1,32 @@
...
-            </option>
-          </select>
-        </div>
-        <div v-if="showUserInput" class="small-8 columns">
+          <option
+            v-for="attribute in filterAttributes"
+            :key="attribute.key"
+            :value="attribute.key"
+          >
+            {{ $t(`FILTER.ATTRIBUTES.${attribute.attributeI18nKey}`) }}
```
Comment (fayazara) [inline app/javascript/dashboard/components/widgets/conversation/components/FilterInput.vue:15]: Small change here to remove this hardcoded value `{{ $t(`FILTER.ATTRIBUTES.${attribute.attributeI18nKey}`) }}` should be replaced with `{{ attribute.name }}` to make it usable. I had raised another PR for this [one](https://github.com/chatwoot/chatwoot/pull/3440/files)

Comment (nithindavid) [reply app/javascript/dashboard/components/widgets/conversation/components/FilterInput.vue:15]: Should I change it here too?

**Review by pranavrajs — APPROVED** [review]
Looks good to me.

File: app/javascript/dashboard/components/widgets/conversation/components/FilterInput.vue (hunk 1 above)
Comment (pranavrajs) [reply app/javascript/dashboard/components/widgets/conversation/components/FilterInput.vue:15]: @fayazara Please resolve this while taking a downmerge.

**Pull request merged**